### PR TITLE
Fix XcodeTask not being able to find podspec

### PR DESCRIPTION
**Pull request by brunobowden**
- Ignore local.properties file from Android Studio
- XML_LEGACY_SETTINGS_IMPORTED, which I presume is a one time import

### CODE REVIEW

**Comment by brunobowden** [comment]
@confile - please take a look

@advayDev1 - the fix is simple, so you can continue making progress elsewhere

File: src/main/groovy/com/github/brunobowden/j2objcgradle/tasks/J2objcXcodeTask.groovy [hunk 1]
```diff
@@ -41,7 +41,7 @@ class J2objcXcodeTask extends DefaultTask {
 
     // CocoaPods podspec file that's used by the Podfile
     @OutputFile
-    File getPodspecFile() { new File(project.buildDir, "podspec") }
+    File getPodspecFile() { new File(project.buildDir, "${project.name}.podspec") }
```
Comment (confile) [inline src/main/groovy/com/github/brunobowden/j2objcgradle/tasks/J2objcXcodeTask.groovy]: Where is ${project.name}.podspec defined in?

**Comment by confile** [comment]
@brunobowden Have a look at my question beside that LGTM

**Comment by brunobowden** [comment]
@confile - it's defined on that line. Not anywhere else, though perhaps it
should be.

**Comment by brunobowden** [comment]
Changed PodName to an @Input method:

    @Input
    String getPodName() { "j2objc-${project.name}" }

**Pull request merged**